proxy/api/src/config.rs

```diff
@@ -21,10 +19,11 @@ pub enum Error {
     MissingExePath(path::PathBuf),
 }
 
-/// Returns the directories to locate all application state.
-#[must_use]
-pub fn dirs() -> ProjectDirs {
-    ProjectDirs::from("xyz", "radicle", "radicle-upstream").expect("couldn't build dirs")
+/// Returns the directories to locate all application state specific to the current identity.
+pub fn id_dirs() -> Result<path::PathBuf, Error> {
+    let home_dir = std::env::var("RAD_HOME").unwrap_or(std::env::var("HOME")?);
+
+    Ok(path::Path::new(&home_dir).join(".radicle/identities/current"))
```

Comment:
Suggestion: There is no immediate benefit but it would be nice to get in the habit of always using `join` to avoid hard-coding platform-specific behavior.

```suggestion
    Ok(path::Path::new(&home_dir).join(&[".radicle", "identities", "current"].iter().collect())
```